lib/hyacinth/adapters/external_identifier_adapter/datacite.README.md

```diff
@@ -0,0 +1,118 @@
+# DataCite DOI implementation
```

Comment:
@fcd1 Thanks for writing up this documentation.  It's great to have.  How about we move this file under a new top level `docs` directory, since I think it could be easy to miss deep down in the hierarchy?

There are some other things, currently in the top level README, that I'm also planning to move under a top level `docs` directory later on.

Reply:
datacite.README.md has been moved over to the top level `docs` directory.